_apidocs/searchgov-results.md

```diff
@@ -99,6 +121,16 @@ Sites indexed via sitemaps or crawling will use the `/i14y` endpoint. Because mo
       | snippet          | Summary of the document
       | publication_date | Publication date of the document (not available on commercial results)
 
+      **If `include_facets=true` in your request**, the following fields may show as well. We only display fields with content, so some web results may not have all fields indicated here.
+
+      | Values       | Description
+      | :--          | :--
+      | content_type | Content type of the document
+      | mime_type    | Mimetype (i.e. file type) of the document
+      | tags         | Tags associated with the document
+      | searchgov_custom1, searchgov_custom2, searchgov_custom3 | Search.gov custom field content associated with the document
```

Comment:
Should we separate these into different rows?

Reply:
updated